specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/2020-01-15-generalAvailability/CommunicationIdentity.json

```diff
@@ -0,0 +1,264 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "contact": {
```

Comment:
let's remove the "contact" element. It is not needed anymore

Reply:
removed